library/trans.c

```diff
@@ -749,7 +847,7 @@ static void *dnet_check_process(void *data)
 
 	while (!n->need_exit) {
 		dnet_check_all_states(n);
-		sleep(1);
+		usleep(10 * 1000); // 10 milliseconds
```

Comment:
In `dnet_check_all_states` we account tries for timeouts. And now timeout is configured by config and equal 3 seconds. Can you fix accounting of tries? See `dnet_net_state::stall`.

Reply:
Thanks, ok, I'll search it.